templates/weavescope/0/docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+weavescope-probes:
+  image: weaveworks/scope:0.11.1
+  privileged: true
+  net: host
+  pid: host
+  labels:
+    io.rancher.scheduler.global: true
+    io.rancher.container.dns: true
+  links:
+    - weavescope-apps:weavescope
+  ports:
+  - "4040:4040"
+  volumes:
+  - "/var/run/docker.sock:/var/run/docker.sock"
+  tty: true
+  command:
+  - "--probe.docker"
+  - "true"
+  - "--no-app"
+  - "weavescope"
```

Comment:
It will be looking for `weavescope` in DNS, how is that supposed to work now, since we separated the app and the probes? ...and is the `weavescope-apps:weavescope` link correct in this context?